_harp/public/pages/git.md

```diff
@@ -1,44 +1,44 @@
-Cuberis use GIT for version control and GitHub for a number of development task. GitHub remote repositories, project development documentation, project issues and development specific discussion. The goal is to keep documentation and discussion in context and project a central location for all information related to development projects. Additionally, deployment of most projects is tied to the Master branch in GitHub in conjunction with DeployHQ.
+<h1 class="page-title">
+  Git & GitHub
+</h1>
 
-For a general introduction to GIT Cuberis has a Codeschool account with excellent video training. That login is available via LastPass.
+## Introduction
 
-Basic online walk through and introduction to GIT and GitHub
+Cuberis uses GIT for version control and GitHub for a number of development tasks, i.e. project development documentation, project issues and development specific discussion. The goal is to keep all documentation and discussion in context and provide a central location for all information related to  projects.
```

Comment:
Sorry to be more nitpicky. Change all occurrences of GIT to **Git**.

Reply:
👍 good call